tools/update-docs-version/src/main/java/com/microsoft/playwright/tools/UpdatePlaywrightVersion.java

```diff
@@ -0,0 +1,53 @@
+package com.microsoft.playwright.tools;
+
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.NoSuchElementException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class UpdatePlaywrightVersion {
+  private static final Pattern VERSION_PATTERN = Pattern.compile(
+    "<groupId>com.microsoft.playwright</groupId>\\s*<artifactId>parent-pom</artifactId>\\s*<version>(\\d+\\.\\d+\\.\\d+)(?:-SNAPSHOT)?</version>"
+  );
+  private static final Pattern MAVEN_PATTERN   = Pattern.compile(
+    "(<dependency>\\s*<groupId>com.microsoft.playwright</groupId>\\s*<artifactId>playwright</artifactId>\\s*)<version>\\d+\\.\\d+\\.\\d+</version>(\\s*</dependency>)"
+  );
+  private static final Pattern GRADLE_PATTERN  = Pattern.compile(
+    "(implementation group: 'com.microsoft.playwright', name: 'playwright', version: )'\\d+\\.\\d+\\.\\d+'"
+  );
+
+  public static void main(String[] args) throws Exception {
+    Path   pomPath    = Paths.get("pom.xml");
+    String pomContent = Files.readString(pomPath, UTF_8);
+
+    Matcher versionMatcher = VERSION_PATTERN.matcher(pomContent);
+    if (!versionMatcher.find()) {
+      throw new NoSuchElementException("Project version was not found");
+    }
+    String version = versionMatcher.group(1);
```

Comment:
Version in pom.xml usually refers to the next development version, which hasn't been published yet. Let's set it to major.(minor-1).0 similar to what we did in https://github.com/microsoft/playwright-java/pull/1748.

Also would be nice to unify this two places, sot that the versions are patched at the same place with the same deps version.